src/ui/index.js

```diff
@@ -9,13 +9,15 @@ import UiBundleCollection from './ui_bundle_collection';
 import UiBundlerEnv from './ui_bundler_env';
 import { UiI18n } from './ui_i18n';
 
-export { uiSettingsMixin } from './ui_settings';
+import { uiSettingsMixin } from './ui_settings';
 
 export default async (kbnServer, server, config) => {
   const uiExports = kbnServer.uiExports = new UiExports({
     urlBasePath: config.get('server.basePath')
   });
 
+  await kbnServer.mixin(uiSettingsMixin);
```

Comment:
Moved here so that the uiExport consumer is registered before we try to consume all of the exports. This bit of pluggability isn't quite as pluggable as originally thought, uiExportConsumers need to be registered between the time that `kbnServer.uiExports` is created and when `kbnServer.uiExports.consumePlugin()` is called for the first time, which doesn't provide any time in this case...